src/test/java/seedu/address/model/person/UniquePersonListTest.java

```diff
@@ -167,4 +167,9 @@ public void asUnmodifiableObservableList_modifyList_throwsUnsupportedOperationEx
         assertThrows(UnsupportedOperationException.class, ()
             -> uniquePersonList.asUnmodifiableObservableList().remove(0));
     }
+
+    @Test
+    public void toStringMethod_producesExpectedOutput() {
```

Comment:
```suggestion
    public void toStringMethod() {
```
In retrospect, this is better. Adding `_producesExpectedOutput` is just stating the obvious. Besides, it does not follow the 3-part naming pattern `methodTested_scenario_expectedBehaivor`
It is fine for the test method to be same as method being tested, unless there are multiple `@Test` methods testing the same method.